crates/router/src/connector/aci/transformers.rs

```diff
@@ -36,12 +39,22 @@ impl TryFrom<&types::ConnectorAuthType> for AciAuthType {
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct AciPaymentsRequest {
+    #[serde(flatten)]
+    pub txn_details: TransactionDetails,
+    #[serde(flatten)]
+    pub payment_method: PaymentDetails,
+    #[serde(flatten)]
+    pub instruction: Option<Instruction>,
```

Comment:
Is there a reason why `#[serde(flatten)]` is being used here? Are these types being reused anywhere else?